docs/guides/package.md

```diff
@@ -121,6 +121,11 @@ hello-world 1.3.1b2 => 1.3.1
 
 ## Publishing your package
 
+!!! note
+
+    A complete guide to publishing from GitHub Actions to PyPI can be found in the
+    [GitHub Guide](integration/github.md#publishing-to-pypi)
+
```

Comment:
The guide could arguably be in either document, it's the intersection of GitHub and building&publishing.